velox/dwio/catalog/fbhive/FileUtils.cpp

```diff
@@ -209,9 +216,13 @@ std::vector<std::pair<std::string, std::string>> FileUtils::parsePartKeyValues(
   return ret;
 }
 
-std::string FileUtils::extractPartitionName(const std::string& filePath) {
+std::string FileUtils::extractPartitionName(
+    const std::string& filePath,
+    bool partitionPathAsLowerCase) {
```

Comment:
Do we need this parameter here? Shouldn't we extract partition name from file path unmodified?

Reply:
@mbasmanova Because the call to makePartName method in [here](https://github.com/facebookincubator/velox/pull/8091/files#diff-1a7bb8270bd257658ec27ab2d73202fa28c9c7a996b571e47a6a8a3709dee75dR225) requires the partitionPathAsLowerCase parameter.

Reply:
@mbasmanova It is OK to me.